tutorials/local-server-api-db-intro.md

```diff
@@ -0,0 +1,71 @@
+# Local Server
+
+## Overview 
+
+In this module we will bring up the API server and do some basic exploration of the database and make an API call against the server. The goal is to help you gain an understanding of how the back end and data is structured and to start gaining an understanding the tools, scripts and languages we use to aid us in development.
+
+## Tasks
+
+### (LOCAL-SERVER-1) Bring up the local server
+
+./project.rb dev-up
+
+This will bring up a server with API endpoints and a local database.
+
+
+#### Dependencies
+You should have the following installed on your machine for this to work
+
+- Ruby installed
+- docker installed and running
+- docker-compose
+- docker-sync (`gem install docker-sync` or `gem install --user-install docker-sync` or non-root installs)
+
+Having ruby gems installed on your path is a good idea as well - adding this to your profile or rc file will help to find the ruby gems:
+
+```sh
+if which ruby >/dev/null && which gem >/dev/null; then
+  PATH="$(ruby -r rubygems -e 'puts Gem.user_dir')/bin:$PATH"
+fi
+```
+
+### (LOCAL-SERVER-2) Using a REST client (e.g. insomnia) make a request to the local server
+  1. Start with the /v1/profile endpoint (http://localhost:8081/v1/profile)
+  2. What was the response?
+
+### (LOCAL-SERVER-3) Repeat LOCAL-SERVER-2 With a Bearer Token
+
+Generate a bearer token and pass this along to the endpoint
+Ge
+  1. What happened, what was the error code and response now?
+  2. What do you think you need to do in order to log in?
+
+### (LOCAL-SERVER-4) Initial Database Exploration
+
+  Run `./project.rb connect-to-db`
+  This will connect to your local database
+
+  1. Run a query to display all of the tables in the database
+  2. Which tables do you think are involved with the error you encountered in task LOCAL-SERVER-3?
+
+### (LOCAL-SERVER-5) Further DB and Endpoint Exploration
+
+  1. Using SQL, look at the schema of the `user` table
+  2. Insert a row with your test account data to the `user` table
```

Comment:
This sounds like a tedious task that's probably not going to result in the response succeeding, and will put your local server into a bad state. I'm not sure I'm following what the goal is here.

Is there an implicit requirement here that you've already created a user in the _test_ environment, and not locally? If so, this should be spelled out somewhere. There also seems to be an assumption that this is your first time running the API server, and you've never visited the local Workbench UI - is that the case?

I don't recall what exactly will happen when you hit the profile API as an unknown user, but locally it may actually just lazily initialize you, in which case at this point you would actually expect to see a row here.